js/components/accessibilityControls.js

```diff
@@ -147,7 +147,16 @@ AccessibilityControls.prototype = {
     this.controller.moveVrToDirection(false, keyDirectionMap[charCode]); //stop rotation if isKeyDown === false or prevent prev rotation if press a button (isKeyDown === true)
```

Comment:
I'm actually seeing `Message Bus Event: moveVrToDirection` messages on the browser's console when using the keyboard even on videos that are not 360. We should check whether the conditions above are working.

Reply:
\+